setup.py

```diff
@@ -125,7 +125,8 @@ def getsitepackages():
                                      "python" + sys.version[:3],
                                      "site-packages"),
                         os.path.join(prefix, "lib", "site-python"),
-                        os.path.join(prefix, "python" + sys.version[:3], "lib-dynload")]
+                        #os.path.join(prefix, "python" + sys.version[:3], "lib-dynload")
+                        ]
```

Comment:
I commented this, because AFAIK `lib-dynload` is a core Python thing not related to site-packages. If we want to add this dir, better add it last in the list. If you guys agree it should not be here, we can just delete the line.